README.md

```diff
@@ -12,6 +12,23 @@ EDGI is already monitoring tens of thousands of pages and will eventually be mon
 
 You can **track upcoming releases** by exploring our [milestones](https://github.com/edgi-govdata-archiving/web-monitoring/milestones) within the GitHub issue queue.
 
+## :hammer: Technology Used
+
+* [**Heroku.**][heroku] A platform for easily deploying applications.
+* [**Review Apps.**][review-apps] A Heroku feature that deploys code
+  from GitHub pull requests as a disposable app on the web.
+* Javascript. For front-end UI.
```

Comment:
We also use it on the back-end of UI and as CLI scripts in `web-monitoring-versionista-scraper`.